nicegui/elements/scene.js

```diff
@@ -275,6 +275,9 @@ export default {
           undefined,
           (error) => console.error(error)
         );
+      } else if (type == "axes_helper") {
+        mesh = new THREE.AxesHelper(args[0]);
+        mesh.material.transparent = true;
```

Comment:
What is the purpose of setting `transparent = true`? My feeling is that without this setting the axes tend to flicker less while orbiting.

Reply:
To be honest, I tried both options and haven‘t noticed a difference. In the end, I tried to stick to the line and curve materials:

https://github.com/zauberzeug/nicegui/blob/1a47d90b16292375459d10a7438ab2958dc34bee/nicegui/elements/scene.js#L226

But we can also remove it if you perfer

Reply:
Oh I see! If we omit this line, we can't reduce the opacity with `material(..., 0.5)`.
It might be worth thinking about automatically switch this flag based on the current opacity. But that's out of scope of this pull request. Let's merge!